### DISCUSSION TITLE
Android platform deployment

### DISCUSSION THREAD

**Discussion by Blackdarked** [zxch3n/PomodoroLogger#258] [Ideas]
This app by far is one of the best To-do app with Pomodoro implementation I've ever seen. And I would like to see if it could be made available on Android platform, one of which a great deal of users are using everyday.

**Comment by S7venLights** [comment]
I have also been looking for a kanban with time tracking, but I need time tracking on the go and as such also need android. I've only found 1 other kanban with time tracking, it works in a web browser but not great on mobile browsers.
https://github.com/plankanban/planka
https://github.com/stars/S7venLights/lists/time-tracking-with-kanban
